src/lib.rs

```diff
@@ -350,6 +350,24 @@
 //! client.set("users.uniques", 42);
 //! ```
 //!
+//! Cadence also supports using UDS with the `UdsMetricSink`:
```

Comment:
Can you add the same documentation to the README (it's basically a copy/paste of the lib.rs docs)?